lib/dotcom_web/templates/trip_plan/_options.html.eex

```diff
@@ -1,14 +1,17 @@
+<h6>When</h6>
+<%=
+  render "_departure.html", Map.merge(assigns, %{
+    title: format_plan_type_for_title(@query),
+    prefix: "trip-plan-departure",
+    datetime: datetime_from_query(@query)
+  })
+%>
+<h6>Modes</h6>
```

Comment:
Why use `<h6>` here? I'm not sure that headings make more sense here semantically than fieldset labels, but given the structure of this page they'd be `<h2>` if anything. [Heading levels shouldn't skip levels](https://dequeuniversity.com/rules/axe/4.8/heading-order).